Configuration/Geometry/python/GeometryExtended2023D5Reco_cff.py

```diff
@@ -45,6 +45,10 @@
 from Geometry.CSCGeometryBuilder.idealForDigiCscGeometry_cff import *
 from Geometry.DTGeometryBuilder.idealForDigiDtGeometry_cff import *
 
+#timing
```

Comment:
@ianna If you look in dictGeometry2023 you'll see these lines are already there. I updated this file from the dict.